src/sql/parts/query/common/resultSerializer.ts

```diff
@@ -29,6 +29,8 @@ import * as pretty from 'pretty-data';
 import { ISlickRange } from 'angular2-slickgrid';
 import * as path from 'path';
 
+let prevSavePath: string;
```

Comment:
any reason not to make this a static property of the ResultSerializer class rather than a global variable?